server/src/main/java/org/elasticsearch/index/codec/tsdb/ES87TSDBDocValuesFormat.java

```diff
@@ -42,13 +43,21 @@ public class ES87TSDBDocValuesFormat extends org.apache.lucene.codecs.DocValuesF
     static final int TERMS_DICT_REVERSE_INDEX_SIZE = 1 << TERMS_DICT_REVERSE_INDEX_SHIFT;
     static final int TERMS_DICT_REVERSE_INDEX_MASK = TERMS_DICT_REVERSE_INDEX_SIZE - 1;
 
+    final BinaryDVCompressionMode binaryDVCompressionMode;
+
     public ES87TSDBDocValuesFormat() {
+        this(BinaryDVCompressionMode.NO_COMPRESS);
```

Comment:
Should this be change to `COMPRESSED_WITH_LZ4`? Otherwise it doesn't get used outside tests?

Reply:
Good catch :)